plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/project/interceptor/AbstractJavaContentRootInterceptor.java

```diff
@@ -88,30 +88,17 @@ private ContentRoot getSourceType(Node node) {
     }
 
     private String _getSourceFolder(ProjectConfigDto projectConfig, String srcAttribute) {
-        Map<String, List<String>> attributes = projectConfig.getAttributes();
+        final Map<String, List<String>> attributes = projectConfig.getAttributes();
         if (!attributes.containsKey(srcAttribute)) {
-            return "";
+            return null;
```

Comment:
Why do you return "" and null when srcAttribute doesn't have value? 
https://github.com/eclipse/che/pull/523/files#diff-6063db316be1e2733d888f9686ae3133R98